apps/codeforafrica/src/components/NavListItem/index.js

```diff
@@ -0,0 +1,35 @@
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+const NavLinkItemRoot = styled("li")(
+  ({ theme: { breakpoints, typography } }) => ({
+    marginBottom: `${typography.pxToRem(20)}`,
+    listStyle: "none",
+    [breakpoints.up("lg")]: {
+      marginBottom: `${typography.pxToRem(10)}`,
+      listStyle: "none",
+    },
+  })
+);
+
+const NavListItem = React.forwardRef(function NavListItem(props, ref) {
+  const { key, children } = props;
+  return (
+    <NavLinkItemRoot key={key} ref={ref}>
+      {children}
+    </NavLinkItemRoot>
+  );
```

Comment:
```suggestion
  return (
    <NavLinkItemRoot {...props} key={key} ref={ref} />
  );
```
React doesn't pass `key` to components. It only uses it to decide if an item in a list has changed and hence should be re-render or not.

Reply:
This hasn't been addressed.